desktop_version/src/FileSystemUtils.cpp

```diff
@@ -447,11 +447,16 @@ void FILESYSTEM_loadFileToMemory(
 	size_t *len,
 	bool addnull
 ) {
+	PHYSFS_File *handle;
+	PHYSFS_sint64 length;
+	PHYSFS_sint64 success;
+
 	if (SDL_strcmp(name, "levels/special/stdin.vvvvvv") == 0)
 	{
 		// this isn't *technically* necessary when piping directly from a file, but checking for that is annoying
 		static std::vector<char> STDIN_BUFFER;
 		static bool STDIN_LOADED = false;
+		size_t length;
```

Comment:
This now shadows the previous length variable, so it should be renamed.

Reply:
Good catch! Fixed, rebased, force-pushed.